client/src/actions/index.js

```diff
@@ -0,0 +1,11 @@
+import * as api from '../api';
+
+export const getUsers = () => async(dispatch) => {
+    try {
```

Comment:
Please be consistent on code indentation. Use either 2 spaces or a tab-char on every file consistently without mixing them. I would recommend setting up your IDE with eslint.
https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint